spec/controllers/sessions_controller_spec.rb

```diff
@@ -28,6 +28,12 @@
       expect(response).to be_redirect
       expect(response.location).to match /^#{stream_url}\??$/
     end
+
+    it "auto create and login pam user" do
+      post :create, params: {user: {remember_me: "0", username: "pam_user1", password: "123456"}}
+      expect(response).to be_redirect
+      expect(response.location).to match /^#{getting_started_url}\??$/
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.